src/1729Essay.sol

```diff
@@ -100,13 +89,12 @@ contract OneSevenTwoNineEssay is Ownable, ERC721, ERC2981 {
                         Royalty
     //////////////////////////////////////////////////////////////*/
 
-    function updateRoyaltyPercentage(uint96 _newPercentageInBips) public onlyOwner {
-        require(_newPercentageInBips <= MAXIMUM_ROYALTY_PERCENTAGE_IN_BIPS, "New royalty percentage cannot be greater than 15%");
-
-        emit RoyaltyPercentageUpdated(_newPercentageInBips);
-
-        royaltyPercentageInBips = _newPercentageInBips;
-        // _setDefaultRoyalty(receiver, feeNumerator);
+    /// @notice returns the author's address and a fixed 10% royalty
+    function royaltyInfo(uint256 tokenId, uint256 salePrice)
+        external
+        view
+        returns (address receiver, uint256 royaltyAmount) {
+        return(essays[tokenId].author, salePrice / 10);   // Not using SafeMath here as the denominator is fixed and can never be zero
```

Comment:
This is lot simpler than I expected. You could actually put in unchecked if you wanted to but  maybe not good practice for this method, which I think is what you mean, because there's zero chance of an over/underflow and >= Solidity 0.8.0 does the checks already.

Reply:
Not blocking

Reply:
OpenZepplin has a SafeMath package that's used in a lot of examples.  I looked into its implementation of division and it's literally return (a / b) but with a check for division by zero first.  The point of the comment is that if in a future iteration of this someone makes the denominator non-fixed it's a reminder that they may at that point need SafeMath or their own check.  Just a hint to future self.